### PR TITLE
feat(frontend): Cookie consent toast

### PR DESCRIPTION
**Pull request by nkrmr**
- Added cookie contest on all pages
- Added js-cookie library to manage cookie
- Setted cookieConsent var with one year expiration date

### CODE REVIEW

File: packages/code-du-travail-frontend/src/common/CookieConsent.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        onRemove={() => {
+          setCookieConsentHidden(true);
+          cookie.set("cookieConsent", true, { expires: 365 });
+        }}
+      >
+        En poursuivant votre navigation sur ce site, vous acceptez l’utilisation
+        de cookies pour établir des mesures de fréquentation et d’utilisation du
+        site.
+        <A
+          title="consulter l'ordonance"
```
Comment (revolunet) [inline packages/code-du-travail-frontend/src/common/CookieConsent.js]: erreur de titre ?

File: packages/code-du-travail-frontend/src/common/CookieConsent.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  }, []);
+  if (isCookieConsentHidden) return null;
+  return (
+    <Wrapper>
+      <Toast
+        variant="info"
+        wide
+        shadow
+        animate="from-bottom"
+        onRemove={() => {
```
Comment (lionelB) [inline packages/code-du-travail-frontend/src/common/CookieConsent.js]: we should use useCallback here

File: packages/code-du-travail-frontend/src/common/CookieConsent.js (hunk 1 above)
Comment (nkrmr) [reply packages/code-du-travail-frontend/src/common/CookieConsent.js]: Mis à jour avec le titre "Mentions Légales"

File: packages/code-du-travail-frontend/src/common/CookieConsent.js (hunk 2 above)
Comment (nkrmr) [reply packages/code-du-travail-frontend/src/common/CookieConsent.js]: Done

File: packages/code-du-travail-api/src/server/routes/__tests__/__snapshots__/items.test.js.snap [hunk 3]
```diff
@@ -56,7 +56,7 @@ Object {
     Object {
       "_id": "2",
       "_index": "cdtn_document_test",
-      "_score": 22.295755,
+      "_score": 22.58624,
```
Comment (lionelB) [inline packages/code-du-travail-api/src/server/routes/__tests__/__snapshots__/items.test.js.snap]: if you don't update api, there is no need to update api score can you check you ES version (7.4.1)

**Comment by nkrmr** [comment]
Closed to test API issue

**Pull request closed without merging**